src/pages/resourcesPage/resourcesPage.js

```diff
@@ -1,51 +1,42 @@
 import React, { useState, useEffect } from "react"
 import "./resourcesPageStyles.css"
 
-export default function UsersPage() {
+export default function ResourcesPage() {
 
-  const [resourceTable, setData] = useState([]);
-  // const [error, setError] = useState("");
-  // const [loading, setLoading] = useState(false);
-  useEffect(() => {
-    const token = localStorage.getItem('token');
-    const userId = localStorage.getItem("userId");
-    const getData = async () => {
-      try {
-        const response = await fetch(
-          `http://localhost:3000/api/resources`, {
-          method: "get",
-          headers: {
-            'Content-type': 'application/json',
-            'authorization': `Bearer ${token}`,
-            'userId': JSON.parse(userId)
-          }
+  const [resourcesData, setResourcesData] = useState([]);
+  
+    useEffect(() => {
+        const token = localStorage.getItem('token');
+        const userId = localStorage.getItem("userId");
+        if(token) {
+            const getResourcesData = async () => {
+                try {
+                const response = await fetch(
+                    `http://localhost:3000/api/resources`, {
+                    method: "get",
+                    headers: {
+                    'Content-type': 'application/json',
+                    'authorization': token,
+                    'userId': JSON.parse(userId)
```

Comment:
why are we sending user id's in all the API calls? Isn't token enough ?

Reply:
checking for the token available in local storage that belongs to that user